Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c

```diff
@@ -0,0 +1,95 @@
+//Floyd Warshall Algorithm using C
+
+#include<stdio.h>
+
+//Vertices count in the Graph
+#define A 4
+
+//Defining INF(infinite) as a large value.
+//It's used for non-connected vertices
+#define INF 999999
```

Comment:
use INT_MAX for infinity

Reply:
Ma'am, I previously used macro, but one of the mentor told not to use and I was having squash problems.
So, shall I change it to macro ?